public/app/plugins/datasource/cloudwatch/components/MetricsQueryEditor/MetricsQueryEditor.test.tsx

```diff
@@ -173,9 +181,6 @@ describe('QueryEditor', () => {
 
         expect(screen.getByText('Label')).toBeInTheDocument();
         expect(screen.queryByText('Alias')).toBeNull();
-        expect(screen.getByLabelText('Label - optional')).toHaveValue(
-          "Period: ${PROP('Period')} InstanceId: ${PROP('Dim.InstanceId')}"
-        );
```

Comment:
```suggestion
        expect(screen.getByText("Period: ${PROP('Period')} InstanceId: ${PROP('Dim.InstanceId')}")).toBeInTheDocument();
```

This should work after passing `preparedQuery.label` to `DynamicLabelsField`.

Reply:
See comment above

Reply:
The text should be visible now. The mock still renders the text in a `<pre>` tag so getting by text should work.